providers/amazon/src/airflow/providers/amazon/aws/hooks/athena_sql.py

```diff
@@ -119,7 +119,7 @@ def conn_config(self) -> AwsConnectionWrapper:
                     raise ValueError(
                         f"Encountered non-JSON in `extra` field for connection {self.aws_conn_id!r}."
                     )
-            except AirflowNotFoundException:
+            except (AirflowNotFoundException, RuntimeError):
```

Comment:
This kind of catch is for backward compatibility in different Airflow versions

Reply:
In this case do we need it -- won't it import AirflowNotFouncException from the right place?

What error does a get_connection throw when the conn is not found?